mars/oscar/backends/router.py

```diff
@@ -37,7 +37,7 @@ def get_instance() -> "Router":
 
     @staticmethod
     def get_instance_or_empty() -> "Router":
-        return Router._instance or _empty_router
+        return Router._instance or Router(list(), None)
```

Comment:
Why remove`_empty_router`?

Reply:
`_empty_router` is global and hold references of previous tests such as `_curr_external_addresses`/`_local_mapping`/`_mapping`/`_cache` if we use same driver.
BTW, I think global state should be avoided as much as possible.

Reply:
On ray, this will cause later tests crash